jets-bench/benches/elements/data_structures.rs

```diff
@@ -160,19 +160,21 @@ impl SimplicityEncode for SimplicityCtx8 {
         let buf = var_len_buf_from_slice(&self.buffer[..buf_len], 8).unwrap();
         let len = Value::u64(self.length as u64);
         // convert to 32 byte array
-        let arr = self
+        let arr: [u8; 32] = self
             .h
             .iter()
             .flat_map(|x| x.to_be_bytes())
-            .collect::<Vec<u8>>();
-        let mid_state = Value::u256_from_slice(&arr);
+            .collect::<Vec<u8>>()
+            .try_into()
+            .unwrap();
+        let mid_state = Value::u256(&arr);
```

Comment:
In 010e675ecf7e7990d3699b5f4fbe4c0c72302a9b:

I guess it doesn't matter because we intend to drop this code anyway after #239 but it would be nice to get rid of the unnecessary `Vec` here.

Reply:
I switched to using `Value::power_of_two`, which is also internally called by `Value::u256`. We still have to allocate a `Vec`, like on master, but we no longer have to create an extra byte array.